locale/en/corteza-server/auth/oauth2-authorize-client.yaml

```
@@ -11,4 +11,4 @@ template:
   errors:
     invalid-user: Cannot continue with unauthorized email, visit <a data-test-id="link-redirect-to-profile" href="{{link}}">your profile</a> and resolve the issue.
 alerts:
-  denied: cannot authorize {{client}}, no permissions
+  denied: Failed to authorize {{client}}. No permissions
```

Think we should go more in the lines of

```suggestion
   failed to authorize {{client}}: insufficient permissions
```